ql/src/experimental/CWE-640/EmailGood.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"net/http"
+	"net/smtp"
+)
+
+// smtp sendMail good
+func mail(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func mailGood(w http.ResponseWriter, r *http.Request) {
```

To avoid a naming conflict when adding this file to the tests.